README.md

```diff
@@ -331,6 +337,7 @@ VAULTENV_RETRY_ATTEMPTS:        9
 VAULTENV_LOG_LEVEL:             Error
 VAULTENV_USE_PATH:              True
 VAULTENV_MAX_CONCURRENT_REQUESTS: 8
+VAULTENV_OVERWRITE_DUPLICATE_VARIABLES: False
```

Comment:
This name seems misleading: when it's true, vaultenv doesn't actually overwrite variables (since it takes the existing values going by the comments), and when it's false, it also doesn't overwrite variables (since it errors).

Reply:
This was fixed with moving to a more explicit choice instead of a boolean.